rj_gameplay/rj_gameplay/play/defensive_clear.py

```diff
@@ -43,6 +44,7 @@ def tick(
         role_results = play.unflatten_results(flat_results)
 
         # Get list of all skills with assigned roles from tactics
+        skills: List[tactic.SkillEntry] = []
         skills = self.two_mark.tick(world_state, role_results[self.two_mark])
```

Comment:
```suggestion
        skills += self.two_mark.tick(world_state, role_results[self.two_mark])
```
line 47 inits an empty list of SkillEntries but then this line resets it to the return type of a tactic's tick(), should change to += instead